openmdao/core/explicitcomponent.py

```diff
@@ -36,6 +38,7 @@ def __init__(self, **kwargs):
         super(ExplicitComponent, self).__init__(**kwargs)
 
         self._inst_functs = {name: getattr(self, name, None) for name in _inst_functs}
+        self._has_derivs = overrides_method('compute_partials', self, ExplicitComponent)
```

Comment:
What if they define `compute_jacvec_product` but not `compute_partials`?

Reply:
Ok, looks like it doesn't matter for the places where it is used.

Reply:
I changed the name of the flag to _has_compute_partials to make things clearer.